cnxeasybake/tests/rulesets/string_args.css

```
@@ -0,0 +1,11 @@
+div {
+  string-set: maybe-has-child '';
```

Would it make sense to make `''` different so that the test can exercise all 3 cases (string-set, overridden string-set, and a default when the string is undefined): 

```suggestion
  string-set: maybe-has-child '[INITIAL]';
```

---

Oh nice. Good thought. This makes sense to test all cases.